examples/license-classifications.yml

```diff
@@ -3,377 +3,338 @@
 # https://github.com/nexB/scancode-toolkit/commit/ed644e4
 #
 # To demonstrate how one can insert a custom written offer
-# include_source_code_offer_in_notice_file has been set to
+# include-source-code-offer-in-notice-file has been set to
 # true for all licenses of the GPL family.
 categories:
 - name: "copyleft"
 - name: "strong-copyleft"
 - name: "copyleft-limited"
 - name: "permissive"
+  description: "Licenses with permissive obligations."
 - name: "public-domain"
+- name: "include-in-notice-file"
+  description: >
+    This category is checked by templates used by the ORT report generator. The licenses associated with this
+    category are included into NOTICE files.
+- name: "include-source-code-offer-in-notice-file"
+  description: >
+    A marker category that indicates that the licenses assigned to it require that the source code of the packages
+    needs to be provided.
+
```

Comment:
Nit: Superfluous blank line.

Reply:
The blank line should separate the sections for the categories and the categorizations. I don't think that it is superfluous.

Reply:
Fine with me.